### PR TITLE
moving modules to wdl

### PR DESCRIPTION
**Pull request by gavin-peng**
https://jira.oicr.on.ca/browse/GRD-656 wdl completed and validated

### CODE REVIEW

**Comment by pruzanov** [comment]
Please update CHANGELOG as well

**Review by pruzanov — CHANGES_REQUESTED** [review]
CHANGELOG needs to be updated

**Comment by gavin-peng** [comment]
CHANGELOG updated

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,7 @@
 # Unreleased
```
Comment (pruzanov) [inline CHANGELOG.md]: This "Unreleased" record is not needed (or is it  misplaced?)

**Review by pruzanov — APPROVED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (gavin-peng) [reply CHANGELOG.md]: Removed. yes, not needed.

**Pull request merged**